addon-test-support/fill-in.js

```diff
@@ -13,8 +14,19 @@ import wait from 'ember-test-helpers/wait';
 */
 export function fillIn(selector, text) {
   let el = getElementWithAssert(selector);
+
+  if (!isFocusable(el)) {
+    throw new Error('Unable to fill element');
```

Comment:
this is located outside of `run()` in order to have an ability to catch exceptions